attributes/alibaba.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+cred_dir = ::File.expand_path(::File.join('~', '.alibaba'))
+credential_file = ::File.join(cred_dir, 'credentials')
```

Comment:
I think we should probably move the alibaba credentials into the same data_bag that has the Azure, GCP, and AWS creds.

Reply:
I have used the same format as the data bag so it should just be a case of copying and paste from what I have. I do not know where the data bag for those is to update.

To get this working I used a local version of the data bag that kitchen could read.